### PR TITLE
Alignment multichannel demo with other samples

### PR DESCRIPTION
**Pull request by antonrom23**

### CODE REVIEW

**Comment by vmatveic** [comment]
@Wovchena , could you please take a look? or add someone responsible for this demo to approve. Thanks.

File: demos/multichannel_demo/fd/main.cpp [hunk 1]
```diff
@@ -56,7 +56,7 @@ void showUsage() {
     std::cout << "    -d \"<device>\"                " << target_device_message << std::endl;
     std::cout << "    -nc                          " << num_cameras << std::endl;
     std::cout << "    -bs                          " << batch_size << std::endl;
-    std::cout << "    -n_ir                        " << num_infer_requests << std::endl;
+    std::cout << "    -nireq                        " << num_infer_requests << std::endl;
```
Comment (IRDonch) [inline demos/multichannel_demo/fd/main.cpp]: Please restore the alignment here.

File: demos/multichannel_demo/hpe/README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Options:
     -d "<device>"                Optional. Specify the target device for a network (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The demo looks for a suitable plugin for a specified device.
     -nc                          Optional. Maximum number of processed camera inputs (web cameras)
     -bs                          Optional. Batch size for processing (the number of frames processed per infer request)
-    -n_ir                        Optional. Number of infer requests
+    -nireq                        Optional. Number of infer requests
```
Comment (IRDonch) [inline demos/multichannel_demo/hpe/README.md]: ... and here.

File: demos/multichannel_demo/hpe/main.cpp [hunk 3]
```diff
@@ -72,7 +72,7 @@ void showUsage() {
     std::cout << "    -d \"<device>\"                " << target_device_message << std::endl;
     std::cout << "    -nc                          " << num_cameras << std::endl;
     std::cout << "    -bs                          " << batch_size << std::endl;
-    std::cout << "    -n_ir                        " << num_infer_requests << std::endl;
+    std::cout << "    -nireq                        " << num_infer_requests << std::endl;
```
Comment (IRDonch) [inline demos/multichannel_demo/hpe/main.cpp]: ... and here.

File: demos/multichannel_demo/fd/main.cpp (hunk 1 above)
Comment (antonrom23) [reply demos/multichannel_demo/fd/main.cpp]: Done

File: demos/multichannel_demo/hpe/README.md (hunk 2 above)
Comment (antonrom23) [reply demos/multichannel_demo/hpe/README.md]: Done

File: demos/multichannel_demo/hpe/main.cpp (hunk 3 above)
Comment (antonrom23) [reply demos/multichannel_demo/hpe/main.cpp]: Done

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**